samples/cpp/hello_query_device/main.cpp

```diff
@@ -117,7 +117,11 @@ int main(int argc, char* argv[]) {
             for (auto&& metricName : supportedMetrics) {
                 if (metricName != METRIC_KEY(SUPPORTED_METRICS) && metricName != METRIC_KEY(SUPPORTED_CONFIG_KEYS)) {
                     std::cout << "\t\t" << metricName << " : " << std::flush;
-                    printParameterValue(core.get_metric(device, metricName));
+                    if (metricName == GPU_METRIC_KEY(MAX_BATCH_SIZE)) {
+                        std::cout << "The result depends on the model size" << std::endl;
```

Comment:
Maybe we can set default value to 1 and return it for empty model as well to avoid any custom processing of this option?

Reply:
I could not find better way not corrupting user application with exception throwing. So I updated the PR with your suggestion. Also I updated the doc too.